lib/sample_rate_converter/Kconfig

```diff
@@ -27,8 +27,16 @@ config SAMPLE_RATE_CONVERTER_FILTER_TEST
 	  intended to be used for unittests only, as they give predictable and easy to understand
 	  output.
 
+config SAMPLE_RATE_CONVERTER_FILTER_SMALL
+	bool "Include the small sample rate converter filters"
+	help
+	  Includes the small filters for the sample rate converter. The small filters uses a small
+	  amount of space and time for the conversion, while also giving some low-pass filter
+	  capabilities.
+
 config SAMPLE_RATE_CONVERTER_MAX_FILTER_SIZE
```

Comment:
Do we need this KConfig at all?
The preprocessor knows the filter sizes so we may use` MAX(ARRAY_SIZE(filter_A), ARRAY_SIZE(filter_B), ...) `

Reply:
Using `MAX()` would be great, but I couldn't find a way to get this to work as filters can be `#ifdef`ed away. If you have some ideas we could sit down and discuss offline!